harvard-university-of-worcester.csl

```diff
@@ -25,7 +25,7 @@
   </macro>
   <macro name="author">
     <names variable="author">
-      <name and="symbol" delimiter-precedes-et-al="never" delimiter-precedes-last="never" et-al-min="19" et-al-use-first="19" initialize-with="." name-as-sort-order="all"/>
+      
```

Comment:
You're right, I didn't mean to delete the whole line. Sorry! I'll add it back in.